circle.yml

```diff
@@ -110,7 +92,7 @@ step-library:
       run:
         name: Build Example
         command: |
-          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=11.4,name=iPhone 6 Plus' -project MapboxNavigation.xcodeproj -scheme Example clean build | xcpretty
+          xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=12.1,name=iPhone 6 Plus' -project MapboxNavigation.xcodeproj -scheme Example clean build | xcpretty
```

Comment:
Should we also upgrade the device model here, or do we want to keep testing on lower-end simulators just in case?

Reply:
I think it makes sense to upgrade. However, most preferably, let's keep it on 6 till we start experiencing problems. Or what do you think?

Reply:
I'm up for dropping Xcode 9 support but we have to test on the minimum deployment target including the base target. You can use [xcode-install](https://github.com/KrauseFx/xcode-install) to install an older simulator. I guess we should run tests, or at least build the example app for iOS 9 and 12.